packit.yaml

```diff
@@ -3,7 +3,10 @@ upstream_package_name: cockpit-podman
 downstream_package_name: cockpit-podman
 actions:
   post-upstream-clone: make cockpit-podman.spec
-  create-archive: make dist
+  # tarball for releases (copr_build, koji, etc.)
+  create-archive:
+    - sh -exc "curl -L -O https://github.com/cockpit-project/cockpit-podman/releases/download/${PACKIT_PROJECT_VERSION}/${PACKIT_PROJECT_NAME_VERSION}.tar.xz"
```

Comment:
This is dreadful...

Is that really the recommended approach?

Reply:
It's certainly not the final solution for all this, that stuff is all fairly new still. I was talking this over in #packit earlier on:

> TomasTomecek[m] | we could even do some of that on our side if more people would use that workflow

Reply:
To see what happens, and for a proper packit issue, I [added a commit to drop create-archive](https://github.com/martinpitt/cockpit-podman/commit/4f22ebfe6cbf6fe01022c4fcfb61134ae19603eb), did a [47.3 release](https://github.com/martinpitt/cockpit-podman/releases/tag/47.3), and it actually [did succeed in building a srpm](https://copr.fedorainfracloud.org/coprs/martinpitt/test-fixes/build/4406887/). However, it cheated again:

    Command: git archive --output cockpit-podman-47.3.tar.gz --prefix cockpit-podman-47.3/ HEAD

so that's not what we want. I'll file a packit issue to behave better by default, and link it from here.

Reply:
I filed a long-winded https://github.com/packit/packit-service/issues/1505 and will add a reference to it.

Reply:
also replying here; sorry Martin I only realized just now: but Packit actually retrieves the archive from the URL defined in the spec file instead of using `create-archive` - at minimum we should properly document that and log that create-archive is not being utilized

as I'm seeing this I'm surprised create-archive is triggered, which is a bug IMO so the issue is completely valid